spec/features/publish/courses/editing_a_levels_tda_course_spec.rb

```diff
@@ -295,4 +301,18 @@ def then_i_am_on_the_add_another_a_level_subject_page
   def and_i_see_the_success_message_that_i_added_an_a_level
     expect(page).to have_content('You have added a required A level')
   end
+
+  def then_i_am_on_the_consider_pending_a_level_page
+    expect(page).to have_current_path(
+      publish_provider_recruitment_cycle_course_a_levels_consider_pending_a_level_path(
+        @provider.provider_code,
+        @provider.recruitment_cycle_year,
+        @course.course_code
+      )
+    )
+  end
+
+  def then_i_see_an_error_message_for_the_consider_pending_a_level_page
+    expect(page).to have_content('Select if you will consider candidates with pending A levels')
```

Comment:
In general, we always expect to see the error on the page twice -- once in the summary and once on the field. So I prefer to have
```suggestion
    expect(page).to have_content('Select if you will consider candidates with pending A levels').twice
```

Reply:
Done :) ♻️